tests/phpunit/Process/Runner/MutationTestingRunnerTest.php

```diff
@@ -147,8 +147,8 @@ public function test_it_applies_and_run_the_mutations(): void
     {
         $mutations = [
             $mutation0 = $this->createMutation(0),
-            $mutation1 = $this->createMutation(1),
-            $mutation2 = $this->createMutation(2, 1000.0),
+            $mutation1 = $this->createMutation(1, 99.0),
+            $mutation2 = $this->createMutation(2, 100.0),
```

Comment:
could you please explain what is this change about? I mean I see that it's time, but why was it needed?